### DISCUSSION TITLE
Where's the documentation

### DISCUSSION THREAD

**Discussion by blakadder** [make-all/tuya-local#894] [Ideas]
I'm trying to implement a number of Tuya devices but its difficult to implement uncommon features because there is no real documentation on various options.

I believe adding documentation explaining how the devices yaml options relate to final configuration and explaning f.e. `mapping:`

**Comment by make-all** [comment]
The configuration options are documented in [custom_components/tuya_local/devices/README.md](https://github.com/make-all/tuya-local/blob/main/custom_components/tuya_local/devices/README.md)
